src/chunks/vocab.rs

```diff
@@ -154,6 +154,16 @@ where
         }
     }
 
+    /// Get the vocab's indexer.
+    pub fn indexer(&self) -> &I {
+        &self.indexer
+    }
+
+    /// Get the lower and upper bound of the generated ngram lengths.
```

Comment:
Either the constructor should also take a tuple or we should have `min_n()` and `max_n()` methods. Or, perhaps even better now that we have recent Rust versions, `RangeInclusive`.

Reply:
For now two separate methods. If we want to use `RangeInclusive` we should do that in a separate PR since there's quite a few places where this would show up.